extensions/liveshare/src/providers/queryProvider.ts

```diff
@@ -132,6 +136,12 @@ export class QueryProvider {
 			}]);
 		};
 
+		let notifyConnectionUriChanged = (ownerUri: string): Thenable<void> => {
+			return self._sharedServiceProxy.request(constants.notifyConnectionUriChangedRequest, [{
```

Comment:
This doesn't look correct - you should be using `.notify` instead of request and the name should be the actual name of the notification (which should be `connectionUriChanged`, not `notifyConnectionUriChanged`)

Reply:
I'm following with what is being done with rebuildIntellisenseCache, which used the request format, I'll try to switch to notify instead and see if it breaks anything.

Reply:
Also it needs to match the name of the azdata method call otherwise it will complain the function does not exist for queryprovider. I will need to change the function back to connectionUriChanged for the dataprotocol otherwise this won't work.

Reply:
the specific error is: Object literal may only specify known properties and connectionurichanged does not exist in type QueryProvider. So either we call it connectionUriChanged or notifyConnectionUriChanged in both sqlops and here, but it cannot be different.